src/etc/inc/util.inc

```diff
@@ -720,6 +720,28 @@ function is_subnetv6($subnet)
     return (is_subnet($subnet) == 6);
 }
 
+/**
+ * @return true if $value is a comma separated list of valid dotted IPv4
+ * addresses, or false otherwise.
+ *
+ * Spaces around the IPv4 addresses are allowed.
+ */
+function is_ipaddrlistv4($value)
```

Comment:
Can we avoid this and just move to handle this in interfaces.php? The MVC framework supports CSV natively so we would want to avoid increasing "library" load for functions that are never going to be reused.

Reply:
Thank you for the feedback!

I could easily move this to `interfaces.inc` but not to `interfaces.php` since the function is used from `interfaces.inc` too. Does that sound good?

> The MVC framework supports CSV natively so we would want to avoid increasing "library" load for functions that are never going to be reused.

Could you elaborate on what you mean by the MVC framework supporting CSV natively? Is there a better way to check that something is a comma separated list of IPv4 addresses than the `explode()` and loop solution below?

Reply:
No extra functions, no validation in interfaces.inc -- this is far too much effort than useful. I've mentioned the MVC pages because all static PHP pages are going to be converted at some point so most extra effort on static PHP pages will either lead to unnecessary work or perhaps also regressions.

Reply:
> no validation in interfaces.inc 

to clarify the old code does this but it should not

Reply:
Updated